### PR TITLE
feat: migrate to schema org

### PR DESCRIPTION
**Pull request by tlgimenes**
This PR uses schema.org and adds vtex/shopify integrations. More info in the loom:
https://www.loom.com/share/be5e1e4ba84747e9ad1b962b4b1f1f6c

Depends on https://github.com/deco-cx/live/pull/72

### CODE REVIEW

File: components/SKUSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import type { Product } from "$live/std/commerce/types.ts";
+
+interface Props {
+  product: Product;
+}
+
+// Navigates the user the the choosen sku
+const onChange = {
+  onchange: "(function(e){ location.pathname = e.target.value; })(event)",
```
Comment (lucis) [inline components/SKUSelector.tsx:9]: haha nice

File: functions/vtexProductListingPage.ts [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  const pageInfo = { hasNextPage: Boolean(pagination.next.proxyURL) };
+  const filters = facets
+    .map((f) => toFilter(f))
+    .filter((x): x is Filter => Boolean(x));
+
+  return {
+    data: {
+      breadcrumb: {
+        "@type": "BreadcrumbList",
+        itemListElement: [],
```
Comment (lucis) [inline functions/vtexProductListingPage.ts:65]: TODO?

File: import_map.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "imports": {
     "$components/": "./components/",
     "$sections/": "./sections/",
-    "$live/": "https://deno.land/x/live@0.3.19/",
+    "$live/": "../live/",
```
Comment (lucis) [inline import_map.json:5]: Change

**Review by lucis — APPROVED** [review]

File: functions/vtexProductListingPage.ts (hunk 2 above)
Comment (tlgimenes) [reply functions/vtexProductListingPage.ts:65]: Uhum, I think so

File: components/SKUSelector.tsx (hunk 1 above)
Comment (tlgimenes) [reply components/SKUSelector.tsx:9]: gotta remove all JS!

**Pull request merged**